EightMinuteEmpireLegends/Core/GameBuilderException.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "../pch.h"
+
+class GameBuilderException : public std::exception
```

Comment:
I think you meant to delete this file. Seems to be a duplicate of the one in Core/GameCore dir